app/networkapi/templates/admin/login.html

```diff
@@ -27,7 +27,7 @@
 
 
 <form action="{{ app_path }}" method="post" id="login-form">
-    {% if social %}
+    {% if backends %}
```

Comment:
"actually work", but as a better answer it checks for the `backends` context variable that the social auth core library adds. If social signin is used, this context variable will exist. If it isn't, then this context variable doesn't exist and the template doesn't render the whole "use google or login with user/pass" bit